benchflow-dsl/src/main/scala/cloud/benchflow/dsl/dockercompose/DockerComposeYamlBuilder.scala

```diff
@@ -0,0 +1,67 @@
+package cloud.benchflow.dsl.dockercompose
+
+import cloud.benchflow.dsl.definition.types.bytes.Bytes
+import cloud.benchflow.dsl.definition.types.bytes.BytesYamlProtocol._
+import net.jcazevedo.moultingyaml.{ YamlString, YamlValue, _ }
+
+/**
+ * @author Jesper Findahl (jesper.findahl@gmail.com)
+ *         created on 2017-05-25
+ */
+class DockerComposeYamlBuilder(dockerComposeYamlValue: YamlValue) {
+
+  val ServicesKey = YamlString("services")
+  val MemLimitKey = YamlString("mem_limit")
```

Comment:
This is what I could find for docker-compose `v2` (https://docs.docker.com/compose/compose-file/compose-file-v2/#restart). 
I think it would be good to discuss  which version we want to support.